forge-core/src/main/java/forge/card/CardRulesPredicates.java

```diff
@@ -546,108 +535,55 @@ public PredicateSplitType(final CardSplitType type) {
+    public static final Predicate<CardRules> IS_EQUIPMENT = CardRulesPredicates.subType("Equipment");
+    public static final Predicate<CardRules> IS_LAND = CardRulesPredicates.coreType(true, CardType.CoreType.Land);
+    public static final Predicate<CardRules> IS_BASIC_LAND = subject -> subject.getType().isBasicLand();
+    public static final Predicate<CardRules> NOT_BASIC_LAND = subject -> !subject.getType().isBasicLand();
+    /** Matches only Plains, Island, Swamp, Mountain, or Forest. */
+    public static final Predicate<CardRules> IS_TRUE_BASIC_LAND = subject -> !subject.getName().equals("Wastes")&&subject.getType().isBasicLand();
+    /** Matches any card except Plains, Island, Swamp, Mountain, or Forest. */
+    public static final Predicate<CardRules> NOT_TRUE_BASIC_LAND = subject -> !subject.getType().isBasicLand() || subject.getName().equals("Wastes");
+    public static final Predicate<CardRules> IS_NONBASIC_LAND = subject -> subject.getType().isLand() && !subject.getType().isBasicLand();
+    public static final Predicate<CardRules> CAN_BE_COMMANDER = CardRules::canBeCommander;
+    public static final Predicate<CardRules> CAN_BE_PARTNER_COMMANDER = CardRules::canBePartnerCommander;
+    public static final Predicate<CardRules> CAN_BE_OATHBREAKER = CardRules::canBeOathbreaker;
+    public static final Predicate<CardRules> CAN_BE_SIGNATURE_SPELL = CardRules::canBeSignatureSpell;
+    public static final Predicate<CardRules> IS_PLANESWALKER = CardRulesPredicates.coreType(true, CardType.CoreType.Planeswalker);
+    public static final Predicate<CardRules> CAN_BE_TINY_LEADERS_COMMANDER = CardRulesPredicates.IS_LEGENDARY.and(CardRulesPredicates.IS_CREATURE.or(CardRulesPredicates.IS_PLANESWALKER));
+    public static final Predicate<CardRules> CAN_BE_BRAWL_COMMANDER = CardRulesPredicates.IS_LEGENDARY.and(CardRulesPredicates.IS_CREATURE.or(CardRulesPredicates.IS_PLANESWALKER));
+    public static final Predicate<CardRules> IS_BATTLE = CardRulesPredicates.coreType(true, CardType.CoreType.Battle);
+    public static final Predicate<CardRules> IS_INSTANT = CardRulesPredicates.coreType(true, CardType.CoreType.Instant);
+    public static final Predicate<CardRules> IS_SORCERY = CardRulesPredicates.coreType(true, CardType.CoreType.Sorcery);
+    public static final Predicate<CardRules> IS_ENCHANTMENT = CardRulesPredicates.coreType(true, CardType.CoreType.Enchantment);
+    public static final Predicate<CardRules> IS_NON_CREATURE_SPELL =
+            CardRulesPredicates.IS_SORCERY
+                    .or(CardRulesPredicates.IS_INSTANT)
+                    .or(CardRulesPredicates.IS_PLANESWALKER)
+                    .or(CardRulesPredicates.IS_ENCHANTMENT) //TODO: Battles? Is testing these one by one really the best way to check "non-creature"?
+                    .or(CardRulesPredicates.IS_ARTIFACT.and(CardRulesPredicates.IS_CREATURE.negate()));
```

Comment:
Per the TODO, I believe Battles should be added here? Though it may be better to just rewrite this as something derived from the type rather than maintaining a list of them here.

Reply:
Why not just use `CardRulesPredicates.IS_CREATURE.negate()` for non-creature-spell?

Reply:
this is a good catch as the current form has become outdated, leading to some chance that card type ratios from deck generation aren't reaching the desired values (or depending on the higher level logic at least cause more looping)

Imo we could go with NonCreature + NonLand + `!isVariant()` just to be safe